docs/nddata/mixins/ndslicing.rst

```diff
@@ -78,9 +77,6 @@ attribute will be sliced too::
     >>> ndd_sliced.uncertainty  # doctest: +FLOAT_CMP
     StdDevUncertainty([1.41421356, 1.73205081])
 
-    >>> ndd_sliced.wcs  # doctest: +FLOAT_CMP
-    array([1., 1.])
-
 ``unit`` and ``meta``, however, will be unaffected.
 
 If any of the attributes are set but do not implement slicing, an info will be
```

Comment:
Apparently the change we are making here really is breaking -- the old behavior was "if I can't slice it, just pass it along to the user with a log message"

Reply:
(nothing to be changed here, just noting it)